Policies.md

```diff
@@ -9,6 +9,7 @@ The WHATWG is guided by [Principles](./Principles.md) of openness, utility, effi
 WHATWG participants operate under a number of policies:
 
 * [Intellectual Property Rights Policy](./IPR%20Policy.md)
+* [Privacy Policy](./Privacy%20Policy.md)
```

Comment:
Should this say Website Privacy Policy? Or should we drop Website below? (It also talks about non-Website matters such as email for instance.)

Reply:
I don't think we plan to put together another privacy policy for non-website stuff, but "website" could also be interpreted as the whole "web presence" of WHATWG. I'm OK with the ambiguity but, if pressed, would lean toward dropping "Website" to allow the policy to have the wider-coverage it assumes.

Reply:
Resolving this as there's now a concrete suggestion to change the title.